learningequality/studio
# add cronjob that clears old tasks every midnight

## Summary
### Description of the change(s) you made

Create a cronjob that spawns a pod that clears tasks older than 7 days.

### Manual verification steps performed
1. Create cronjob on my local machine that connects to develop
2. Run that cronjob manually
3. Check that there are no jobs older than 7 days


### Reviewer's Checklist
#### This section is for reviewers to fill out.
